src/main/scala/cromwell/binding/ContinueOnReturnCode.scala

```diff
@@ -0,0 +1,13 @@
+package cromwell.binding
+
+sealed trait ContinueOnReturnCode {
+  def continueFor(returnCode: Int): Boolean
```

Comment:
I'm a little torn here as I prefer using pattern matching over polymorphism for these sorts of things, but where it's used would add one of those nested layers of matching which I also hate.

One way people handle these things polymorphically in ADTs is to not do the override but have a single function in the trait which then does the match. I won't call it more idiomatic but it's definitely closer to the FP side of the fence in terms of who sides w/ it.

So it'd be something like this in the trait:

    def continueFor(returnCode: Int): Boolean = this match {
      case x: ContinueOnReturnCodeSet => x.returnCodes.contains(returnCode)
      case ContinueOnReturnCodeFlag => 
    }

That's probably not 100% right. This pattern works because this match has exhaustiveness checking. Whether or not it's better to put the logic in the trait vs scattered across the instances, well that's a debate in the community in general.